data/map planets.txt

```diff
@@ -4574,6 +4646,15 @@ planet "Typhon Station"
 		threshold 2000
 		fleet "Large Militia" 8
 
+planet "Ut Divitas"
+	attributes "incipias quarg" quarg
+	landscape land/canyon13
+	description "The surface is bare and there is no sign of life to be seen anywhere. A few lonely but giant cities sprawl like flowers on the surface of the moon, and the architecture is abstract and elegant like nothing humans have ever built."
```

Comment:
Why would there be no sign of life anywhere, if there are Quarg working in the spaceport?
```suggestion
	description "The surface here is bare and there is no sign of life to be seen anywhere. A few lonely but giant cities sprawl like flowers on the surface of the moon, and the architecture is abstract and elegant unlike anything humans have ever built."
```